docs/fork-clone-pullrequest.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Fork, Clone and Pullrequest
+keywords: homepage
+sidebar: talk_sidebar
+permalink: /fork-clone-pullrequest.html
+summary:
+---
+
+Fork/Clone 
```

Comment:
Where would you include them? Do you have any suggestions? 
By the way, would there be a way to add a link to my Medium post instead?https://medium.com/@MichMach/contribution-to-oss-2a24447770c9  
If yes, where would you suggest to add it?